src/main/java/core/basesyntax/db/TransactionList.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Transaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionList {
```

Comment:
I don't think we need to store TransactionList(lets just store data about fruits), lets just get list of transactions from parser and process them